akka-actor-typed-tests/src/test/java/akka/actor/typed/javadsl/ActorLoggingTest.java

```diff
@@ -0,0 +1,74 @@
+package akka.actor.typed.javadsl;
+
+import akka.actor.typed.Behavior;
+import akka.event.Logging;
+import akka.japi.pf.PFBuilder;
+import akka.testkit.CustomEventFilter;
+import akka.testkit.typed.TestKit;
+import com.typesafe.config.ConfigFactory;
+import org.junit.AfterClass;
+import org.junit.Test;
+import org.scalatest.junit.JUnitSuite;
+import scala.concurrent.duration.FiniteDuration;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
```

Comment:
... in wrong place

Reply:
Maybe I should fix that new-file-template after manually moving it in every new file for a year...